disnake/widget.py

```diff
@@ -185,21 +186,22 @@ def __init__(
         self.muted: Optional[bool] = data.get("mute", False) or data.get("self_mute", False)
         self.suppress: Optional[bool] = data.get("suppress", False)
 
-        try:
-            game = data["game"]
-        except KeyError:
-            activity = None
-        else:
-            activity = create_activity(game, state=state)
+        # overwrite base type's @property since widget members always seem to have `avatar: null`,
+        # and instead a separate `avatar_url` field with a full url
+        self.avatar: Optional[Asset] = None
+        if avatar_url := data.get("avatar_url"):
+            self.avatar = Asset(state, url=avatar_url, key=avatar_url, animated=False)
 
-        self.activity: Optional[Union[BaseActivity, Spotify]] = activity
+        self.activity: Optional[Union[BaseActivity, Spotify]] = None
+        if activity := (data.get("activity") or data.get("game")):
```

Comment:
pretty sure it can never be game.

Reply:
https://github.com/DisnakeDev/disnake/pull/736#discussion_r967536618 also applies here